wireguard-install.sh

```diff
@@ -123,13 +123,17 @@ function installWireGuard() {
 		apt-get update
 		apt-get install -y wireguard iptables resolvconf qrencode
 	elif [[ ${OS} == 'debian' ]]; then
-		if ! grep -rqs "^deb .* buster-backports" /etc/apt/; then
+		if [[ ${PRETTY_NAME} == "Debian GNU/Linux 10 (buster)" ]]; then
+			if ! grep -rqs "^deb .* buster-backports" /etc/apt/; then
 			echo "deb http://deb.debian.org/debian buster-backports main" >/etc/apt/sources.list.d/backports.list
-			apt-get update
-		fi
+			apt update
```

Comment:
```suggestion
			apt-get update
```
Scripts are supposed to use apt-get not apt.